docs/docs/testing-your-assistant.mdx

```diff
@@ -117,113 +117,142 @@ Here are some examples:
+Since training is not completely deterministic, the whole process is repeated
+three times for each configuration specified.
+A graph with the mean and standard deviations of
+[f1-scores](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html)
+across all runs is plotted.
 
 The f1-score graph, along with all train/test sets, the trained models, classification and error reports,
 will be saved into a folder called `nlu_comparison_results`.
 
-### Intent Classification
+Inspecting the f1-score graph can help in understanding if you have enough data for your NLU model.
+If the graph shows that f1-score is still improving when all of the training data is used,
+it may improve further with more data. But if f1-score has plateaued when all training data is used,
+adding more data may not help.
+
+If you want to change the number of runs or exclusion percentages, you can:
+
+```bash {3}
+rasa test nlu --nlu data/nlu.yml
+  --config config_1.yml config_2.yml
+  --runs 4 --percentages 0 25 50 70 90
+```
+
+### Interpreting the Output
+#### Intent Classifiers
 
-The `rasa test` script will produce a report, confusion matrix, and confidence histogram for your model.
+The `rasa test` script will produce a report, confusion matrix, and confidence histogram for your intent classification model.
 
-The report logs precision, recall and f1 measure for each intent and entity,
+The report logs [precision](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html),
+[recall](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.recall_score.html) and
+[f1-score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html) for each intent,
 as well as providing an overall average. You can save these reports as JSON files using the `--report` argument.
 
-The confusion matrix shows you which intents are mistaken for others;
-any samples which have been incorrectly predicted are logged and saved to a file called `errors.json` for easier debugging.
+The confusion matrix shows which intents are mistaken for others.
+Any samples which have been incorrectly predicted are logged and saved to a file called `errors.json` for easier debugging.
 
-The histogram that the script produces allows you to visualise the confidence distribution for all predictions, with the volume of correct and incorrect predictions being displayed by blue and red bars respectively. Improving the quality of your training data will move the blue histogram bars to the right and the red histogram bars to the left of the plot.
+The histogram allows you to visualize the confidence for all predictions,
+with the correct and incorrect predictions being displayed by blue and red bars respectively.
+Improving the quality of your training data will move the blue histogram bars to the right and the
+red histogram bars to the left of the plot.
```

Comment:
and will hopefully lead to less red, right? 🙂

Reply:
Added that bit.